woocommerce/class-sv-wc-framework-bootstrap.php

```diff
@@ -278,7 +278,7 @@ public function render_update_notices() {
 				// describe the way to fix it
 				echo '<p>';
 					printf(
-						/** translators: Placeholders: %1$s - <a> tag, %2$s - </a> tag, %3$s - <em> tag, %4$s - </em> tag, %5$s - <a> tag, %6$s - </a> tag, %7$s - <a> tag, %8$s - </a> tag */
+						/* translators: Context is for plugins to activate or deactivate tags. HTML Placeholders: %1$s - <a> tag, %2$s - </a> tag, %3$s - <em> tag, %4$s - </em> tag, %5$s - <a> tag, %6$s - </a> tag, %7$s - <a> tag, %8$s - </a> tag */
```

Comment:
The word `tags` here seems to be a leftover from refactoring, unless I am missing something?

```suggestion
						/* translators: Context is for plugins to activate or deactivate. HTML Placeholders: %1$s - <a> tag, %2$s - </a> tag, %3$s - <em> tag, %4$s - </em> tag, %5$s - <a> tag, %6$s - </a> tag, %7$s - <a> tag, %8$s - </a> tag */
```

Reply:
typo on my end